.github/workflows/conflict-labeler.yml

```
@@ -0,0 +1,20 @@
+name: Check Merge Conflicts
+
+on:
+  push:
+    branches:
+      - master
+  pull_request_target:
+
+jobs:
+  Label:
+    if: github.actor != 'PJBot'
+    runs-on: ubuntu-latest
```

There's no PJBot in this repo

```suggestion
  Label:
    runs-on: ubuntu-latest
```

---

I figured that it wouldn't hurt to have it regardless, but if you like it better without then that's ok